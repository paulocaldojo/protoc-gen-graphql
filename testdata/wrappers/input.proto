syntax = "proto3";

package protoc_gen_graphql.test.wrappers;

option go_package = "github.com/paulocaldojo/protoc-gen-graphql/testdata/wrappers";

import "google/protobuf/wrappers.proto";

message Wrappers {
  google.protobuf.BoolValue bool_value = 1;
  google.protobuf.BytesValue bytes_value = 2;
  google.protobuf.DoubleValue double_value = 3;
  google.protobuf.FloatValue float_value = 4;
  google.protobuf.Int32Value int32_value = 5;
  google.protobuf.Int64Value int64_value = 6;
  google.protobuf.StringValue string_value = 7;
  google.protobuf.UInt32Value uint32_value = 8;
  google.protobuf.UInt64Value uint64_value = 9;

  repeated google.protobuf.BoolValue bool_values = 10;
}
