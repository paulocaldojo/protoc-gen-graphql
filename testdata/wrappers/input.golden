# DO NOT EDIT! Generated by protoc-gen-graphql.

type ProtocGenGraphqlTestWrappers_Wrappers {
  boolValue: Boolean!
  bytesValue: String!
  doubleValue: Float!
  floatValue: Float!
  int32Value: Int!
  int64Value: Float!
  stringValue: String!
  uint32Value: Float!
  uint64Value: Float!
  boolValues: [Boolean!]!
}

input ProtocGenGraphqlTestWrappers_WrappersInput {
  boolValue: Boolean
  bytesValue: String
  doubleValue: Float
  floatValue: Float
  int32Value: Int
  int64Value: Float
  stringValue: String
  uint32Value: Float
  uint64Value: Float
  boolValues: [Boolean!]
}
